docs/experiment/general/experiment-duration-estimate.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Experiment Duration Estimate
+description: Understand how experiment duration estimate is calculated
+---
+
+!!! info
+    This is only for the primary metric and sequential testing right now. This is not supported in Experiment Results.
+
+Given the current statistics (control/treatment mean, control/treatment variance, and control/treatment exposures), Amplitude forecasts what will happen in the future and sees how many days it will take to reach statistical significance. Note if any of these statistics changes a lot the estimate will be off. If there are multiple variants we take the max of the estimates. As Amplitude gets more data, the prediction will be better. Note that the estimate will change over time. 
+
+## FAQ
+
+### 1. Why is the duration estimate not showing?
+
+The duration estimate only shows up when the follow criteria is met:
+
+1. All the statistics assumptions are met
+    1. The absolute lift is outside the confidence interval
+    2. The confidence interval is flipped (lower confidence interval > upper confidence interval)
+    3. The standard error is very small
+    4. Variance is negative
+    5. Conversion rate > 1 or < 0 (if applicable)
+2. The metric has already reached statistical significance
+3. The end date of the analysis window is in the past
+4. There are enough observations. We use the [same condition](https://help.amplitude.com/hc/en-us/articles/4403176829709-How-Amplitude-Experiment-uses-sequential-testing-for-statistical-inference#why-don%E2%80%99t-i-see-any-confidence-interval-on-the-confidence-interv) as for the starting to compute the confidence interval. 
+
+### 2. What does worst case, average case, and best case mean?
+
+The time until a hypothesis test reaches statistical significance is a random variable. We use the worst case, average case, and best case to indicate the uncertainty in our estimate. The best case estimate of 3 days means that 20% of the time the experiment will reach stat sig in <=3 days. The average case estimate of 7 days means that 50% of the time the experiment will reach stat sig in <=7 days. The worst case estimate of 10 days means that 80% of the time the experiment will reach stat sig in <=10 days.
+
+### 3. Why is the largest duration estimate 40 days?
```

Comment:
this 40 days cap may change in the future. Should I still include this?

Reply:
I think one way to rephrase the question is "Is there a cap for the duration estimate that we calculate?" and the answer could be "Currently we put a cap at 40 days for the following reasons..."